Basic-Car-Maintenance/Shared/Dashboard/Views/AddMaintenanceView.swift

```diff
@@ -31,6 +33,21 @@ struct AddMaintenanceView: View {
                          comment: "Maintenance event title text field header")
                 }
                 
+                Section {
+                    Picker(selection: $selectedVehicle) {
+                        ForEach(vehicles) { vehicle in
+                            Text(vehicle.name)
+                        }
+                    } label: {
+                        Text("Select a vehicle",
+                             comment: "Maintenance event vehicle picker label")
+                    }
+                    .pickerStyle(.menu)
+                } header: {
+                    Text("Vehicle",
+                         comment: "Maintenance event vehicle picker header")
+                }
+                
```

Comment:
This isn't keeping track of the vehicle that's selected. See video

https://github.com/mikaelacaron/Basic-Car-Maintenance/assets/22946902/c3da09b6-762f-409b-8a5b-dc43765b2d2b